docs/UseCases.rst

```diff
@@ -0,0 +1,36 @@
+=====================
+Use Cases & Philosopy
+=====================
+
+.. warning:: Incorrect Information below
+
+    Until this document is reviewed and corrected, it is known to contain errors, especially w.r.t. philosopy.
+
+Philosopy
+=========
+
+Frost is intended to build upon the pytest framework to provide extra features to support "infrastructure auditing".
+
+The added features include:
+
+    - providing a standard way to inject meta data in the JSON test result file.
+    - providing a CLI wrapper (the ``frost`` program) to simplify execution of tests within the full supported frost environment.
+
+Note that the added features are all extensions to the normal pytest operations. Therefore, all tests can be executed directly from pytest. This can be highly useful for adhoc queries, and the development of new tests.
+
+Target Domain
+-------------
+
+Frost is intended to simplify the process of auditing existing system configurations, when the configuration data is accessible via an API. Frost assumes that the test results (in JSON format) will be acted upon by a subsequent non-Frost process.
```

Comment:
>Frost assumes that the test results (in JSON format) will be acted upon by a subsequent non-Frost process.

I'm not sure if it is accurate to say that "Frost assumes this". This is true, but Frost just outputs a standardized json report.

Might be more accurate to say something like "Frost is solely scoped to auditing controls and providing a report of the current state"

Reply:
mmm - I was thinking "frost the system", but I see it's confusing. The JSON report is one of the big customizations w.r.t. basic pytest. Let me scratch my head a bit here.

Reply:
@ajvb [rewrite](https://github.com/hwine/frost/compare/hwine%2Fusecases%5E...hwine%2Fusecases?diff=unified#diff-5fe763154d28cda87f636430568dc6849423e32e791ec62e08d75bbd1171b81aL24-L25) any better?